front/app/components/admin/GraphCards/ReactionsByTimeCard/index.tsx

```diff
@@ -25,7 +25,7 @@ import {
 
 type Props = ProjectId & Dates & Resolution;
 
```

Comment:
I'm getting weird type errors in this component (especially the `xlsx` prop on line 54. Are you getting those too? Might be something weird in my setup because I really don't get the error

Reply:
No, I'm not seeing anything in this component with my setup